Included more details for new users how to get started (#56)
FAIRmat-NFDI/nomad-distro-template: README.md

    cd nomad-distro-template
    ```

3. Change the images in the `docker-compose.yaml` and `configs/nomad.yaml`
   The images should be created after the initalization of your repository. Look under packages.

>    In the `docker-compose.yaml`, change the URL wherever there is `image: ghcr.io/fairmat-nfdi/nomad-distro-template:main` to where your image is found. The same goes for the jupyter-image.

Reviewer: This should really not be needed? They should be changed as part of the repository initialization. Could you link me the repo where you had an issue with this?

Author: You are right.
The issue was here: https://github.com/FAU-LAP/nomad-distro-LAP/tree/development
It is already resolved now, however.
The initialize workflow ran again, when I created a new branch, and I merged the changes from that branch into main.

Reviewer: Okay, we already have a big warning about this at the very top so if possible I would avoid repeating this again:
![image](https://github.com/user-attachments/assets/806ab5eb-eff5-4914-8cce-43a78cdc1fef)

We could however add something to FAQ at the bottom if you still think it's unclear?

   
   

4. _On Linux only,_ recursively change the owner of the `.volumes` directory to the nomad user (1000)

    ```sh
    sudo chown -R 1000 .volumes
    ```

5. Pull the images specified in the `docker-compose.yaml`

    Note that the image needs to be public or you need to provide a PAT (see "Important" note above).

    ```sh
    docker compose pull
    ```

6. And run it with docker compose in detached (--detach or -d) mode

    ```sh
    docker compose up -d
    ```

7. Optionally you can now test that NOMAD is running with

    ```
    curl localhost/nomad-oasis/alive
    ```

8. Finally, open [http://localhost/nomad-oasis](http://localhost/nomad-oasis) in your browser to start using your new NOMAD Oasis.

#### Updating the image

> 1. To update the image, just push your changes to GitHub and the workflow will automatically create the new images.

Reviewer: This section is about updating the local image and not generating a new image. That is instead described in the "Adding a Plugin" step. We could however add a line on what causes such an update to happen.
```suggestion
Any pushes to the main branch of this repository, such as when [adding a plugin](#adding-a-plugin), will trigger a pipeline that generates a new app and jupyter image.
```

> 2. Whenever you update your image you need to shut down NOMAD using

Reviewer: ```suggestion
1. To update your local image you need to shut down NOMAD using
```

    ```sh
    docker compose down
    ```

    and then repeat steps 4. and 5. above.
   

> 3. You can remove unused images to free up space by running

Reviewer: ```suggestion
2. You can remove unused images to free up space by running
```

    sudo chown -R 1000 .volumes
    ```

> 5. Pull the images specified in the `docker-compose.yaml`

Reviewer: ```suggestion
4. Pull the images specified in the `docker-compose.yaml`
```

    Note that the image needs to be public or you need to provide a PAT (see "Important" note above).

    ```sh
    docker compose pull
    ```

> 6. And run it with docker compose in detached (--detach or -d) mode

Reviewer: ```suggestion
5. And run it with docker compose in detached (--detach or -d) mode
```

    ```sh
    docker compose up -d
    ```

> 7. Optionally you can now test that NOMAD is running with

Reviewer: ```suggestion
6. Optionally you can now test that NOMAD is running with
```

    ```
    curl localhost/nomad-oasis/alive
    ```

> 8. Finally, open [http://localhost/nomad-oasis](http://localhost/nomad-oasis) in your browser to start using your new NOMAD Oasis.

Reviewer: ```suggestion
7. Finally, open [http://localhost/nomad-oasis](http://localhost/nomad-oasis) in your browser to start using your new NOMAD Oasis.
```
